### PR TITLE
#731: Fixing screenshot

### PR DESCRIPTION
**Pull request by hani-ibrahim**
fixing #731

### CODE REVIEW

**Comment by hani-ibrahim** [comment]
👍

**Comment by hani-ibrahim** [comment]
👍

File: AtlasUI/AtlasUI/Utils/UIBuilder.swift [hunk 1]
```diff
@@ -21,33 +21,41 @@ extension UIBuilder {
         configureView()
         configureConstraints()
 
-        if let view = self as? UIView {
-            view.buildSubviews()
-        } else if let viewController = self as? UIViewController {
-            viewController.view.buildSubviews()
-        }
+        let view = (self as? UIView) ?? (self as? UIViewController)?.view
```
Comment (bonkey) [inline AtlasUI/AtlasUI/Utils/UIBuilder.swift]: Smells a little, but not more than parmigiano ;-)

Comment (hani-ibrahim) [inline AtlasUI/AtlasUI/Utils/UIBuilder.swift]: I thought it's simpler than 2 conditions 😼

**Comment by hani-ibrahim** [comment]
👍

**Comment by bonkey** [comment]
👍

**Comment by bonkey** [comment]
@hani-ibrahim `testPaymentButtonWithPaymentSuccess` [failed on buddybuild 4 times, on all but iPhone 4s/9.3](https://dashboard.buddybuild.com/apps/57a305cb34a9450100595b71/build/58a5c3270324560100623370#tests)

**Pull request merged**